### DISCUSSION TITLE
Select/deselect All in View / Visibility

### DISCUSSION THREAD

**Discussion by BoneheadFraggle** [home-assistant/frontend#7501] [Configuration]
When editing Visibility on a view, it would be awesome with a master toggle to select/deselect all users.

I have just changed 10 views x 18 users to only show it for one user each. That rendered a lot of clicks :-)

**Comment by matthiasdebaat** [comment]
Could you draw a small mockup of what you've got in mind?

**Comment by BoneheadFraggle** [comment]
Limited to Paint at the moment so please look beyond the optics, but something like this:

![image](https://user-images.githubusercontent.com/2287810/130659666-1fb374cd-8574-4e98-93c8-786ef27c34f5.png)

Much like the Header Toggle in an ordinary card.

Maybe called "Toggle All/None"?
